### ISSUE TITLE
ImfHeader.h:22:10: fatal error: ImathVec.h: No such file or directory

### ISSUE THREAD

**Issue by sboehmann** [AcademySoftwareFoundation/openexr#1043]
I am trying to update from 2.5.6 to 3.0.4. OpenEXR 3.0.4 builds, but my application no longer does. The error is:

```
.../include/OpenEXR/ImfHeader.h:22:10: fatal error: ImathVec.h: No such file or directory
   22 | #include "ImathVec.h"
      |          ^~~~~~~~~~~~
```

The reason is that I don't have `<prefix>/include/OpenEXR` and `<prefix>/include/Imath` in my include paths, only `<prefix>/include`.

**Comment by simogasp** [comment]
Imath provides both `<prefix>/include/` and `<prefix>/include/Imath` as include directories, so it should be able to see the header

**Comment by lgritz** [comment]
Yes, the fully qualified `#include <Project/header.h>` is always preferred in any context, IMHO.

**Comment by ianmasters** [comment]
Is this ever going to be fixed? Including without the Imath/ prefix does not work with VS2022 and vcpkg manifest mode. I have to fix up the header files whenever they are refreshed. If there's a better way or a way to include the Imath into the project's include directories I'd love to know. I've tried relative paths etc. but no luck.

**Comment by hermanhermitage** [comment]
@ianmasters suspect you are "doing it wrong".  working fine here.

**Comment by meshula** [comment]
Per @lgritz's comment, a PR implementing this change would be welcome.

**Comment by cary-ilm** [comment]
#2376

**Comment by cary-ilm** [comment]
Culling old issues. I submitted #2376 to resolve this, for the next major/minor release.

**Comment by cary-ilm** [comment]
Closing this as resolved by #2376
